test/test_half_float_ops.py

```diff
@@ -0,0 +1,45 @@
+
+import numpy as np
+import torch
+
+from caffe2.python import core, workspace
+from torch.testing._internal.common_utils import TestCase
+
+
+class HalfFloatOpsTest(TestCase):
+
+    def test_half_to_float(self):
+        def get_c2_result(weights):
```

Comment:
I don't think we need to test against C2 for this. Can we add a reference implementation in PT itself?